tests/integration/cmd_namespace_test.go

```diff
@@ -196,7 +196,7 @@ ComponentSettings:
 					g.Expect(gjson.Get(out, gjsonStr).String()).To(Equal("true"))
 					// ensure that some namespace is marked as "active: false"
 					g.Expect(gjson.Get(out, "namespaces.#[active==false]#.name").String()).ShouldNot(ContainSubstring(commonVar.Project))
-				}).WithTimeout(10 * time.Second).Should(Succeed())
+				}, 10*time.Second, 1*time.Second).Should(Succeed())
```

Comment:
Isn't this already covered by https://github.com/redhat-developer/odo/commit/70c5bd505b67cbb1cb84e90581b6aacc4b1598a1? Or are we still getting issues because the default polling interval is relatively low (10ms)?
The first time I noticed similar flakes was because of the way the test was written: calling `odo list namespace` first, then calling `odo list namespace -o json`, but sometimes the second call would not contain the expected namespace/project.

Anyways, the changes here make sense to me. But I find it more readable to use the `With*` methods. I usually need to think twice to determine which argument is the polling interval and which one is the timeout :D

```suggestion
				}).WithPolling(1*time.Second).WithTimeout(10 * time.Second)Should(Succeed())
```

Reply:
I didn't notice this was a recent change, I noticed we are using withTimeout, and not added value for WithPolling.
Thanks.

Reply:
Whoops, there was a missing `.` in the change suggested :/

Reply:
fixed